tools/setup_riak.escript

```diff
@@ -16,9 +22,9 @@ main(_) ->
         pong ->
             io:format("Riak node ~p~n", [RiakNode]),
             try setup_riak_node(RiakNode)
-            catch Class:Reason ->
+            catch ?STACKTRACE(Class, Reason, Stacktrace)
                 %% This script runs inside michalwski/docker-riak, which still uses OTP 20.3
-                Stacktrace = erlang:get_stacktrace(),
+                %Stacktrace = erlang:get_stacktrace(),
```

Comment:
just removve it

Reply:
Done in the scope https://github.com/esl/MongooseIM/pull/3200